lib/multipart.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const { resolve } = require('path')
+const { readFileSync } = require('fs')
+const FormData = require('form-data')
+
+function getFormData (string) {
+  try {
+    return JSON.parse(string)
+  } catch (error) {
+    try {
+      const path = resolve(string)
+      const data = readFileSync(path, 'utf8')
+      return JSON.parse(data)
```

Comment:
Here I am not allowing `.js` files anymore since it was being a little trickier than plain `.json`.

Reply:
that’s ok!